services/dashboard/scripts/widgets/dag/edit.html

```diff
@@ -1,6 +0,0 @@
-<form role="form">
-  <div class="form-group">
-    <label for="appId">Application ID</label>
-    <input type="text" class="form-control" id="appId" ng-model="config.appId" placeholder="Enter Application">
-  </div>
-</form>
```

Comment:
For the new dashboard we gain breadcrumbs but lose ability to move widgets and edit them. Since this was never listed in dashboard requirements we lose functionality we may need later. Let's update #591 to list requirements that dashboard should have. This way big changes to the dashboard can be better evaluated in terms of features added or removed. Make the requirements list similar to how its done in #27.